docs/installation.rst

```diff
@@ -53,6 +53,44 @@ or for different functionalities of Modin. Here is a list of dependency sets for
 
   pip install "modin[dask]" # If you want to use the Dask backend
 
+Installing from conda-forge
+---------------------------
+
+Modin releases can be installed using Miniconda from conda-forge channel. Starting from 0.10.1
+it is possible to install modin with chosen engine(s). Current options are:
+
++-----------------------------+-------------------------------+
+| Package name in conda-forge | Engine(s)                     |
++-----------------------------+-------------------------------+
+| modin                       | Dask                          |
++-----------------------------+-------------------------------+
+| modin-all                   | Dask, Ray, OmniSci_           |
++-----------------------------+-------------------------------+
+| modin-dask                  | Dask                          |
++-----------------------------+-------------------------------+
+| modin-ray                   | Ray                           |
++-----------------------------+-------------------------------+
+| modin-omnisci               | OmniSci                       |
++-----------------------------+-------------------------------+
```

Comment:
```suggestion
+---------------------------------+-----------------------------------+
| **Package name in conda-forge** | **Engine(s)**                     |
+---------------------------------+-----------------------------------+
| modin                           | Dask                              |
+---------------------------------+-----------------------------------+
| modin-all                       | Dask, Ray, OmniSci_               |
+---------------------------------+-----------------------------------+
| modin-dask                      | Dask                              |
+---------------------------------+-----------------------------------+
| modin-ray                       | Ray                               |
+---------------------------------+-----------------------------------+
| modin-omnisci                   | OmniSci                           |
+---------------------------------+-----------------------------------+
```

Just to highlight first line of the table.

Reply:
Also, I would change the following sentence at top of the file:
```
There are a couple of ways to install Modin. Most users will want to install with pip, but some users may want to build from the master branch on the GitHub repo. The master branch has the most recent patches, but may be less stable than a release installed from pip.
```
to
```
There are a couple of ways to install Modin. Most users will want to install with pip or conda, but some users may want to build from the master branch on the GitHub repo. The master branch has the most recent patches, but may be less stable than a release installed from pip or conda.
```
(added "or conda" two times)

Reply:
Will do, thanks!